### PR TITLE
Theme switcher

### PR DESCRIPTION
**Pull request by ianmah**
gif

### CODE REVIEW

File: src/components/Common.js [hunk 1]
```diff
@@ -41,13 +42,19 @@ export const Button = styled.a`
...
     p.theme.custom_imgs === 'hc' &&
-    css`
+    `
       background: url(${buttonBG});
       background-size: contain;
       background-repeat: no-repeat;
     `}
 `
 
+export const Button = props => (
```
Comment (jackyzha0) [inline src/components/Common.js:52]: what is this change doing exactly?

**Review by jackyzha0 — APPROVED** [review]

File: src/components/Common.js (hunk 1 above)
Comment (ianmah) [reply src/components/Common.js:52]: Buttons without `href` do not change pointer to cursor and do not appear clickable/have good keyboard accessibility. This change is to add a default `href`

Comment (jackyzha0) [reply src/components/Common.js:52]: the more you know! thanks

Comment (ianmah) [reply src/components/Common.js:52]: Actually this is an `a` element. Not exactly sure what happens if it were a `button` elem but what i said is defs true for `a`s

**Pull request merged**